Makefile

```diff
@@ -191,29 +191,39 @@ service-restart-all:
 	@cd $(ROOT_DIRECTORY) && for SERVICE in $(shell ls -1 services) ; do $(MAKE) service-restart SERVICE="services/$${SERVICE}"; done
 	@cd $(ROOT_DIRECTORY) && for SERVICE in migrations tools; do $(MAKE) service-restart SERVICE="$${SERVICE}"; done
 
+GINKGO_FLAGS += --require-suite --poll-progress-after=10s --poll-progress-interval=20s -r # Add flags, e.g. GINKGO_FLAGS="--json-report=report.json" make test
```

Comment:
I'm good with this change, but please move this declaration and that of `GINKGO_CI_FLAGS`, `GOTEST_PKGS`, and `GOTEST_FLAGS` to the top of the file so they are easily visible (like all of the rest currently are). Also, the comment isn't necessary since it seems pretty obvious to me.

Reply:
My preference is to have them there where I can see them without scrolling to the top of the file and back. Preferences differ. I'll change them.